src/m365/spo/commands/site/site-recyclebinitem-restore.ts

```diff
@@ -0,0 +1,79 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  siteUrl: string;
+  id: string;
+}
+
+class SpoSiteRecycleBinItemRestoreCommand extends SpoCommand {
+  public get name(): string {
+    return commands.SITE_RECYCLEBINITEM_RESTORE;
+  }
+
+  public get description(): string {
+    return 'Restores given items from the site recycle bin';
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+    if (this.verbose) {
+      logger.logToStderr(`Restoring items from recycle bin at ${args.options.siteUrl}...`);
+    }
+
+    const requestUrl: string = `${args.options.siteUrl}/_api/site/RecycleBin/RestoreByIds`;
+
+    const ids: string[] = args.options.id.split(',');
```

Comment:
Does this work if values are separated with `, ` (comma + space) or should we trim the values before passing them to the request payload?

Reply:
for sure it will not. We should trim after splitting or just replace(" ", ""). will try to improve this part 👍. Thanks for the very good comment👍

Reply:
added trim after splitting